src/main/java/com/minecolonies/util/StructureWrapper.java

```diff
@@ -564,19 +444,19 @@ public boolean findNextBlockToClear()
                 return false;
             }
         }
-        //Check for air blocks and if blocks below the hut are different from the schematicWorld
-        while ((worldBlockAir() || doesSchematicBlockEqualWorldBlock()) && count < Configurations.maxBlocksCheckedByBuilder);
+        //Check for air blocks and if blocks below the hut are different from the structure
+        while ((worldBlockAir() || doesStructureBlockEqualWorldBlock()) && count < Configurations.maxBlocksCheckedByBuilder);
 
         return true;
     }
 
     private boolean isAirBlock()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "isAirBlock" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUnusedPrivateMethod)